src/core/services/convert.spec.js

```diff
@@ -0,0 +1,89 @@
+import * as Convert from './convert';
+
+describe('Convert', () => {
+	describe('parseText', () => {
+		const parseText = Convert.parseFactory('text');
+
+		it('should parse Number to String', () => {
+			expect(parseText(123)).to.equal('123');
+		});
+		it('should parse Number to String', () => {
+			expect(parseText(-123)).to.equal('-123');
+		});
+		it('should parse Number to String', () => {
+			expect(parseText(-1e04)).to.equal('-10000');
+		});
+		it('should return null if argument is empty', () => {
+			expect(parseText(null)).to.equal(null);
+		});
+		it('should return "undefined" if argument is undefined', () => {
+			expect(parseText(undefined)).to.equal('undefined');
+		});
+	});
+
+	describe('parseNumber', () => {
+		const parseNumber = Convert.parseFactory('number');
+
+		it('should parse String of number to Number', () => {
```

Comment:
please add more incorect and border tests like parseNumber('foo') parseNumber('12foo') parseNumber(Number.MAX...